### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by giomke**
Add well known companies channels

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ We started a new [tools](https://github.com/vitalysim/Awesome-Hacking-Resources/
 ## Table of Contents
  * [Learning the Skills](#learning-the-skills)
  * [YouTube Channels](#youtube-channels)
+   * [Companies](#Companies)
```
Comment (Justincase210) [inline README.md]: Py crypto

File: README.md [hunk 2]
```diff
@@ -17,6 +17,7 @@ We started a new [tools](https://github.com/vitalysim/Awesome-Hacking-Resources/
...
    * [Conferences](#Conferences)
    * [NEWS](#NEWS)
  * [Sharpening Your Skills](#sharpening-your-skills)
```
Comment (Justincase210) [inline README.md]: I need help

**Comment by Justincase210** [comment]
I need to know what to do with it and what is your sLution

**Comment by giomke** [comment]
@Justincase210 
I’m sorry, I didn’t catch that. I don't know "sLution" meaning : ).
This channels contains useful workshops and webinars

**Comment by vitalysim** [comment]
Thanks

**Pull request merged**